### PR TITLE
[PHEE-538] Intergation test code refactoring

### PR DESCRIPTION
**Pull request by dhruvsonagara**
## Description

* [PHEE-538 Intergation test code refactoring](https://fynarfin.atlassian.net/browse/PHEE-538?atlOrigin=eyJpIjoiZDRkZDc2ZjExNTg1NGY2MDgxMmM5NDcwNDUwYWUxNzMiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!
- [x] Followed the PR title naming convention mentioned above.

- [x] Acknowledge that we will not merge PRs that are not passing the checks ("green") - it is your (author's) responsibility to get a proposed PR to pass all the checks, not primarily the project's maintainers.

- [x] The PR title should include a JIRA ticket

- [x] Design-related bullet points or design document links related to this PR added in the description above.

- [x] Updated corresponding Postman Collection or API documentation for the changes in this PR.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [x] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

- [x] Followed coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

FYI our guidelines for code reviews same as https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: src/test/java/org/mifos/integrationtest/cucumber/stepdef/BaseStepDef.java [hunk 1]
```diff
@@ -76,67 +55,68 @@ public class BaseStepDef {
...
     Logger logger = LoggerFactory.getLogger(this.getClass());
 
-    protected static String batchId;
-    protected static String tenant;
-    protected static String response;
-    protected static String request;
-    protected static Integer statusCode;
-    protected static String accessToken;
-    protected static String filename;
+    // protected static String batchId;
```
Comment (fynmanoj) [inline src/test/java/org/mifos/integrationtest/cucumber/stepdef/BaseStepDef.java:63]: remove commented lines

File: src/test/java/org/mifos/integrationtest/cucumber/stepdef/ScenarioScopeDef.java
Comment (fynmanoj) [inline src/test/java/org/mifos/integrationtest/cucumber/stepdef/ScenarioScopeDef.java:1]: rename class to ScenarioScopeState or ScenarioScopeCintext to match its purpose

File: src/test/java/org/mifos/integrationtest/cucumber/stepdef/MockFlowTestDef.java [hunk 2]
```diff
@@ -23,25 +23,29 @@ public class MockFlowTestDef extends BaseStepDef {
     @Autowired
     OperationsAppConfig operationsAppConfig;
 
+    @Autowired
+    ScenarioScopeDef scenarioScopeDef;
```
Comment (fynmanoj) [inline src/test/java/org/mifos/integrationtest/cucumber/stepdef/MockFlowTestDef.java:27]: rename variable to scenarioScopeState or scenarioScopeCintext to match its purpose

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: src/test/java/org/mifos/integrationtest/cucumber/stepdef/BaseStepDef.java (hunk 1 above)
Comment (dhruvsonagara) [reply src/test/java/org/mifos/integrationtest/cucumber/stepdef/BaseStepDef.java]: Done

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**